mezzanine/core/static/mezzanine/js/admin/content_typed.js

```diff
@@ -0,0 +1,28 @@
+jQuery(function($) {
+    var toggleID = function(id) {
+        // Add or remove the page ID from the cookie IDs string.
+        var index = $.inArray(id, ids.split(','));
+        if (index == -1) {
+            if (ids) {ids += ',';}
+            ids += id;
+        }
+        document.cookie = cookie + '=' + ids + '; path=/';
+    };
+
```

Comment:
> The tricky part is how page_tree.js then integrates with that so it can run toggleID - I'm not sure about the best way to do that, and what order things occur in with jQuery when dealing with multiple event handlers, but there must be some sane way to deal with that.

Oddly enough, it seems that all handlers are called regardless of the redirect so the order doesn't matter. That is, if the `location.href` is changed in the first handler called, it doesn't take effect until after the second handler is called.